apps/omg_eth/lib/eth.ex

```diff
@@ -133,11 +132,12 @@ defmodule OMG.Eth do
     signature |> ExthCrypto.Hash.hash(ExthCrypto.Hash.kec()) |> to_hex()
   end
 
-  defp filter_not_removed(logs) do
-    logs |> Enum.filter(&(not Map.get(&1, "removed", true)))
+  defp filter_not_removed(logs, want_removed: want_removed) do
```

Comment:
the function name now does not match, 
maybe use ```:maps.filter``` directly in get_ethereum_events?

Reply:
renamed function + used `match?` for more readable implementation.

I'd rather avoid erlangey calls if possible

Reply:
renamed function + used `match?` for more readable implementation.

I'd rather avoid erlangey calls if possible